taskvine/src/worker/vine_cache.h

```diff
@@ -30,11 +30,19 @@ typedef enum {
 } vine_cache_type_t;
 
 typedef enum {
+	VINE_FILE_UNKNOWN,
```

Comment:
This is looking much better.  However, there is likely to be some confusion of names b/c we already have a number of `VINE_FILE_X` symbols.  Suggest that you call the field `f->status` and the values `VINE_FILE_STATUS_X`